Traceback (most recent call last):
  File "C:/Users/Black/Desktop/python/automate_boring_Stuff/raise_assert_example.py", line 19, in <module>
    raise Exception("This is an error message")
Exception: This is an error message
